docs/docs/60-new-docs/20-quickstart/index.md

```diff
@@ -325,55 +377,9 @@ the previous section.
           direct: true
       promotionTemplate:
         spec:
-          vars:
-          - name: gitopsRepo
-            value: ${GITOPS_REPO_URL}
-          - name: imageRepo
-            value: public.ecr.aws/nginx/nginx
-          - name: srcPath
-            value: ./src
-          - name: outPath
-            value: ./out
           steps:
-          - uses: git-clone
-            config:
-              repoURL: \${{ vars.gitopsRepo }}
-              checkout:
-              - branch: main
-                path: \${{ vars.srcPath }}
-              - branch: stage/\${{ ctx.stage }}
-                create: true
-                path: \${{ vars.outPath }}
-          - uses: git-clear
-            config:
-              path: \${{ vars.outPath }}
-          - uses: kustomize-set-image
-            as: update-image
-            config:
-              path: \${{ vars.srcPath }}/base
-              images:
-              - image: \${{ vars.imageRepo }}
-                tag: \${{ imageFrom(vars.imageRepo).Tag }}
-          - uses: kustomize-build
-            config:
-              path: \${{ vars.srcPath }}/stages/\${{ ctx.stage }}
-              outPath: \${{ vars.outPath }}/manifests.yaml
-          - uses: git-commit
-            as: commit
-            config:
-              path: \${{ vars.outPath }}
-              messageFromSteps:
-              - update-image
-          - uses: git-push
-            config:
-              path: \${{ vars.outPath }}
-          - uses: argocd-update
-            config:
-              apps:
-              - name: kargo-demo-\${{ ctx.stage }}
-                sources:
-                - repoURL: \${{ vars.gitopsRepo }}
-                  desiredRevision: \${{ outputs.commit.commit }}
+          - task:
+              name: promo-process
```

Comment:
For the UI, it might be nice to set an alias here so that the `Promotion` steps become prettier (and do not show `task-0`).

Reply:
Fixed in ade6b4554b8660ef7b4a4b60dbc8230efda5f660